app/Services/Docker/DockerService.php

```
@@ -147,13 +137,12 @@ public function logs(string $service, bool $follow = false, ?callable $callback
         $args[] = $service;
 
-        $process = $this->createDockerComposeProcess($args);
-        $process->setTimeout(null);
+        $composeCommand = $this->buildComposeCommand($args);
 
         if ($callback) {
-            $process->run($callback);
+            Process::timeout(0)->run($composeCommand, $callback);
         } else {
-            $process->run(fn ($type, $buffer): string => print_r($buffer, true));
+            Process::timeout(0)->run($composeCommand, fn ($type, $buffer): string => print_r($buffer, true));
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$type'. (undefined)

(UnusedFormalParameter)